finality-aleph/src/abft/legacy.rs

```diff
@@ -1,3 +1,4 @@
+pub use aleph_primitives::LEGACY_FINALITY_VERSION as VERSION;
```

Comment:
The "same" should happen in the file finality-aleph/src/abft/current.rs

Reply:
Done.